app/components/emoji-selector.hbs

```diff
@@ -35,24 +38,27 @@
             <button
               class="cool-button"
               type="button"
+              title={{t "chat.titles.gif"}}
               {{on "click" this.openGifDialog}}
               >
-              GIF
+              {{t "chat.gif"}}
             </button>
             <button
               class="cool-button fruit-tip-button text-xl font-black"
               type="button"
               style="padding: 0.1em !important;"
+              title={{t "chat.titles.fruits"}}
               {{on "click" this.openFruittipDialog}}
               >
               <span class="flex">
-                <img src="{{this.randomFruitImage}}" style="height: 3ex !important;" >
+                <img src="{{this.randomFruitImage}}" style="height: 3ex !important;" width="33px" height="33px" alt="send fruit" >
```

Comment:
I was confused about this until I realized that I copied and pasted these buttons from the chat into the modal. It seems like this should be ok though @mcfiredrill 
I'm also thinking I should refactor these buttons into some container that would make the code here less repetitive

Reply:
Hmm kind of confused why we need the style 3ex and the width and height values as well? Wouldn't one of them override the other??

Reply:
I'll see what these styles are even doing. Maybe create a CSS class to simplify it too

Reply:
![oversized cabbage](https://user-images.githubusercontent.com/2829501/224517425-76eaba77-510f-461e-bf4a-a2bcb6c69a2f.gif)
lol ok so that's what it's doing